### PR TITLE
NGSTACK-948 added aria label and roles for tags

### PR DESCRIPTION
**Pull request by baljakl**

### CODE REVIEW

**Review by zulicek — APPROVED** [review]

File: translations/messages.en.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ ngsite.layout.sponsored: 'Sponsored'
 ngsite.search.placeholder: 'Search'
 ngsite.search.button: 'Search'
 
+ngsite.tags.tag: 'Tag'
+ngsite.tags.related_tags: "Related Tags"
```
Comment (emodric) [inline translations/messages.en.yaml:16]: `Related tags` (with lowercase `t`)

Comment (baljakl) [reply translations/messages.en.yaml]: adjusted translation for Related tags

File: translations/messages.de.yaml [hunk 2]
```diff
@@ -12,6 +12,9 @@ ngsite.layout.sponsored: 'Gesponsert'
 ngsite.search.placeholder: 'Suche'
 ngsite.search.button: 'Suche'
 
+ngsite.tags.tag: 'Tag'
+ngsite.tags.related_tags: "Verwandte tags"
```
Comment (emodric) [inline translations/messages.de.yaml:16]: Ha, now this is wrong. In German, this actually needs to be uppercase `T` :smile:

Comment (baljakl) [reply translations/messages.de.yaml]: haha, fixed

**Review by emodric — APPROVED** [review]

**Pull request merged**